### DISCUSSION TITLE
Change background colour of PolyStat Panels

### DISCUSSION THREAD

**Discussion by infa-ahossain** [grafana/grafana#71024] [Dashboards, Panels & Visualizations]
Currently, the Grafana PolyStat panel does not have the option to change the background colour.

To make the dashboards look more visually appealing to our customers, we need this feature in the PolyStat.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82596. Feel free to continue the discussion around this there. Thank you!
